### ISSUE TITLE
feature request: dataset downloads (plus file checksums)

### ISSUE THREAD

**Issue by abubelinha** [bionomia/bionomia#341]
Hi again

I'd like to regularly download our institution full datasets by their GBIF datasetId, as described here:
https://bionomia.net/collection-data-managers

As I understand, I need to either search for them in Bionomia web interface or access through `https://bionomia.net/dataset/{datasetId}`
In that url, we have a link to a Zenodo record where we can download the data files.

Does bionomia-api already contain a shortcut way to do it? 
(i.e. method which receives datasetId and returns back the Zenodo record url -at least-).
I couldn't find this among Bionomia api [options](https://bionomia.net/developers/)

Zenodo apis would also let you link individual filenames by record id -so we could just request i.e. `attributions.csv.zip` for a particular `datasetId`-, as long as Bionomia provides something like this:

Base URL: https://api.bionomia.net/dataset.json
?id={datasetID} (required)
&file={attributions.csv.zip} (optional)

Returns .json containing Zenodo urls for all files associated to the corresponding Zenodo record.
Or redirects to Zenodo download for the specific file (when optionally requested)

I think it could be useful.
Thanks a lot.
@abubelinha

**Comment by abubelinha** [comment]
Thanks for moving the link!


You may also want to note a small typo in the Spanish version of the page, under each attributed person:

Collected XXX and identified YYY
Colectó XXX e identificó YYY

("and" is translated to "y" except when the next word starts with "i")


Regarding the hashes/checksums I suppose that will take longer. 
Are you commenting here when done? 
(issue is closed now but I guess it was because you moved the link)

**Comment by dshorthouse** [comment]
Sorry, I closed this through the commit message and had not yet commented. Checksums will be applied when new data packages are regenerated in a week or so & thereafter (after the next GBIF download and re-processing); these are static on the file system, not generated on-the-fly.

Thanks for the grammatical fix. I applied this on https://translate.bionomia.net using Crowdin, which is tied to GitHub and it will eventually submit an automated pull request. All these user cards are cached so, once pulled, I'll have to flush their caches. All this to say, it'll take some time to deploy on production.
